File: python_work/prob/circle.py
from point import Point

class Circle:
    # p=Point()
    def __init__(self,x,y,r):
        print("일로온다.",x,y,r)
        # pass 내용없을때 적음
        self.p=Point(x,y)
        self.r=r

    def showInfo(self):
        self.p.showInfo()
        print(f"반지름={self.r}")

c=Circle(2,2,4)
c.showInfo()

File: java_work/ex0708/src/ex0708/Circle.java
package ex0708;

class Point{
	int xPos, yPos;
	public Point(int xPos, int yPos) {
		this.xPos=xPos;
		this.yPos=yPos;
	}
	public void showPointInfo() {
		System.out.println("["+xPos+","+yPos+"]");
	}
}

public class Circle {
	Point point;
	int r;
	public Circle(int i, int j, int k) {
		point=new Point(i,j);
		this.r=k;
	}
	public void showPointInfo() {
		System.out.println("반지름 r="+this.r);
		point.showPointInfo();
	}
	public static void main(String[] args) {
		Circle circle=new Circle(2,2,4);
		circle.showPointInfo();		
	}
}
